espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/helpers/ViewFinder.kt

```diff
@@ -268,6 +270,9 @@ object ViewFinder {
                 else
                     onView(allOf(isDescendantOfA(`is`(root)), withIndex(matcher, 0)))
                 return listOf(ViewGetter().getView(viewInteraction))
+            } catch (e: AppNotIdleException){
+                throw InvalidElementStateException("The application is expected to be in idle state in order for Espresso to interact with it." +
```

Comment:
The message template could be extracted to a constant to avoid unnecessary duplication